dune/grid/cpgrid/Indexsets.hpp

```diff
@@ -36,7 +36,6 @@ along with OPM.  If not, see <http://www.gnu.org/licenses/>.
 #ifndef OPM_INDEXSETS_HEADER
 #define OPM_INDEXSETS_HEADER
 
-#include <dune/common/nullptr.hh>
```

Comment:
Is this related to changes in recent versions of Dune or simply pruning an unused header?